src/actions/actionBoundText.tsx

```diff
@@ -211,7 +212,7 @@ export const actionWrapTextInContainer = register({
       appState,
     );
     let updatedElements: readonly ExcalidrawElement[] = elements.slice();
-    const containerIds: AppState["selectedElementIds"] = {};
+    const containerIds: Mutable<AppState["selectedElementIds"]> = {};
```

Comment:
Could we add this also elsewhere?

Reply:
you mean this annotation instead of adhoc `{[id: string]: true}`?

Reply:
seems reasonable for the temp object